src/utils/generateArtifacts.ts

```diff
@@ -37,7 +37,7 @@ function decorateEnvrionmentsWithAppId(
   for (const [key, value] of Object.entries(environments)) {
     decoratedEnvrionment[key] = {
       ...value,
-      appId: namehash.hash(value.appName)
+      appId: value.appName && namehash(value.appName)
```

Comment:
Every app should have an appId... so I'm wondering if we shouldn't catch and error in the case of an `appName` not being present?

Reply:
Fixed by https://github.com/aragon/buidler-aragon/pull/115